### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by scsldb**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
update roadmap

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by yikeke** [comment]
5.0 GA 后，5.0 分支上的 roadmap 文件应该可以删掉了，在下一个主版本出来之前，只需要在 master 分支上维护 roadmap 文件

**Review by zz-jason — COMMENTED** [review]
LGTM

File: roadmap.md [hunk 1]
```diff
@@ -9,27 +9,27 @@ aliases: ['/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs-cn/stable/roadmap/','/d
...
-- [ ] 优化延时的抖动 [#18005](https://github.com/pingcap/tidb/issues/18005)
+- [x] `CREATE BINDING` 语句支持绑定 `UPDATE`/`DELETE`/`INSERT` 语句 [#15827](https://github.com/pingcap/tidb/issues/15827)
+- [x] 优化同时执行 DDL 和 DML 语句时悲观事务的模型，提升系统稳定性 [#18098](https://github.com/pingcap/tidb/issues/18098)
+- [x] 优化延时的抖动 [#18005](https://github.com/pingcap/tidb/issues/18005)
 
 ## 提升系统性能，降低系统延时
 
-- [ ] 优化批量删除的性能 [#18028](https://github.com/pingcap/tidb/issues/18028)
-- [ ] 优化系统内存管理，降低系统占用内存的量 [#17479](https://github.com/pingcap/tidb/issues/17479)
+- [x] 优化批量删除的性能 [#18028](https://github.com/pingcap/tidb/issues/18028)
```
Comment (zhangjinpeng1987) [inline roadmap.md:18]: Do you mean coprocessor cache?

File: roadmap.md [hunk 2]
```diff
@@ -49,19 +49,19 @@ aliases: ['/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs-cn/stable/roadmap/','/d
 
 - [ ] 恢复到任意时间点 (PITR) [#325](https://github.com/pingcap/br/issues/325)
 - [ ] 支持修改列的类型 [#17526](https://github.com/pingcap/tidb/issues/17526)
-- [ ] 提升发现及诊断性能问题的效率，降低用户的使用成本 [#18867](https://github.com/pingcap/tidb/issues/18867)
-- [ ] 字符集的排序规则支持 `utf8mb4_unicode_ci` 和 `utf8_unicode_ci` [#17596](https://github.com/pingcap/tidb/issues/17596)
+- [x] 提升发现及诊断性能问题的效率，降低用户的使用成本 [#18867](https://github.com/pingcap/tidb/issues/18867)
```
Comment (zhangjinpeng1987) [inline roadmap.md:52]: Seems not finished?

File: roadmap.md (hunk 1 above)
Comment (scsldb) [reply roadmap.md:18]: #### 提升`delete * from table where id < ? limit ?` 语句执行的性能，p99 性能提升了 4 倍

File: roadmap.md (hunk 2 above)
Comment (scsldb) [reply roadmap.md]: 已改

File: roadmap.md (hunk 1 above)
Comment (zhangjinpeng1987) [reply roadmap.md:18]: 内部就是 cop cache

**Review by zhangjinpeng1987 — APPROVED** [review]
LGTM

**Review by yikeke — APPROVED** [review]

**Pull request merged**